tests/storage/__init__.py

```diff
@@ -203,8 +203,9 @@ def test_create_collection(self, requires_collections, get_storage_args,
         if getattr(self, 'dav_server', '') == 'radicale':
             pytest.skip('MKCOL is broken under Radicale 1.x')
 
-        if getattr(self, 'dav_server', '') == 'icloud':
-            pytest.skip('iCloud requires a minimum-length for collection name')
+        if getattr(self, 'dav_server', '') in \
+           ('icloud', 'fastmail', 'davical'):
+            pytest.skip('Manual cleanup would be necessary.')
```

Comment:
I managed to create and delete a calendar on Fastmail from my iPhone (CalDAV). Here's the request from `mitmproxy`:

```
DELETE https://caldav.fastmail.com/dav/calendars/user/hobarrera%40fastmail.com/9B0C98FD-6E99-4684-80FD-3198E43F865D/

Host:             caldav.fastmail.com
Authorization:    Basic REDACTED
Connection:       keep-alive
Accept:           */*
User-Agent:       iOS/10.2.1 (14D27) dataaccessd/1.0
Content-Length:   0
Accept-Language:  en-gb
Accept-Encoding:  gzip, deflate
```